src/widgets/_linked.scss

```diff
@@ -0,0 +1,98 @@
+%linked_not_left {
+    border-top-left-radius: 0;
+    border-bottom-left-radius: 0;
+}
+
+%linked_not_right {
+    border-right-style: none;
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
+}
+
+%linked_not_top {
+    border-top-left-radius: 0;
+    border-top-right-radius: 0;
+}
+
+%linked_not_bottom {
+    border-bottom-style: none;
+    border-bottom-left-radius: 0;
+    border-bottom-right-radius: 0;
+}
+
+.linked:not(.vertical) > button,
+.linked:not(.vertical) > entry {
+    &:dir(ltr) {
+        &:not(:first-child) {
+            @extend %linked_not_left;
+        }
+
+        &:not(:last-child) {
+            @extend %linked_not_right;
+        }
+    }
+
+    &:dir(rtl) {
+        &:not(:first-child) {
+            @extend %linked_not_right;
+        }
+
+        &:not(:last-child) {
+            @extend %linked_not_left;
+        }
+    }
+}
+
+.linked.vertical > button,
+.linked.vertical > entry {
+    &:not(:first-child) {
+        @extend %linked_not_top;
+    }
+
+    &:not(:last-child) {
+        @extend %linked_not_bottom;
+    }
+}
+
+combobox.linked button:nth-child(2) {
```

Comment:
Is there some amazing GTK reason this isn't `:last-child`?

Reply:
Yeah Comboboxes are fuckin weird. I wouldn't mess with it